src/state/slices/common-selectors.ts

```diff
@@ -229,3 +233,17 @@ export const selectHighestMarketCapFeeAsset = createSelector(
     return sortedFeeAssets[0]
   },
 )
+
+export const selectIsCustomAsset = createSelector(selectAssetById, (asset): boolean => {
+  if (!asset) return false
+  return !!asset.isCustomAsset
+})
+
+// This is a specific case we want to check for, where we have a custom asset but no market data
+export const selectIsCustomAssetWithoutMarketData = createSelector(
+  selectMarketDataByAssetIdUserCurrency,
+  selectIsCustomAsset,
+  (marketData, isCustomAsset): boolean => {
+    return !marketData.price && isCustomAsset
```

Comment:
default market data price is `'0'` which always returns false in this selector currently. compare directly against default value
```suggestion
    return marketData.price === '0' && isCustomAsset
```

Reply:
🙏 I noticed this bug yesterday - and you've just found the cause.

Thank you ser: https://github.com/shapeshift/web/pull/7370/commits/9f7c5f1860557b4f526baab4c81aa1cdd6029341

Left both checks in for safety, as it's possible the market data object doesn't exist at all when we check.